rskj-core/src/main/java/co/rsk/net/discovery/PeerDiscoveryRequest.java

```diff
@@ -73,4 +74,14 @@ public boolean validateMessageResponse(InetSocketAddress responseAddress, PeerDi
     public boolean hasExpired() {
         return System.currentTimeMillis() > expirationDate;
     }
+
+    @Override
+    public String toString() {
+        return new ToStringBuilder(this)
+                .append("messageId", this.messageId)
+                .append("message", this.message.toString())
```

Comment:
I wouldn't explicitly call `toString()` here and below - field could be `null` or in some cases its call might not be needed at all

Reply:
Done